easyaudit/migrations/0004_auto_20170620_1354.py

```diff
@@ -11,8 +11,8 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AlterIndexTogether(
-            name='crudevent',
-            index_together=set([('object_id', 'content_type')]),
+        migrations.AddIndex(
+            model_name='crudevent',
+            index=models.Index(fields=['object_id', 'content_type'], name='easyaudit_c_object__82020b_idx'),
```

Comment:
I feel like you either:

1. Shouldn't add the index name
2. Should remove the operation that introduces the index name, migration 0018

Reply:
I'd be good with removing both; I just wanted it to be absolutely consistent. My bullet points in the PR description should hold true on why specifying it doesn't matter that much, though

Reply:
That's actually not both, you just need to pick one.

If you leave it as is, the migration 0018 will break because the index already exists.

This will break test suites that recreate the database from scratch, which is Django's default behavior.

Reply:
Implemented suggestion 1

Reply:
@jourdanrodrigues Turns out `name` is a required arg for `AddIndex`, so had to revert.

Reply:
It makes sense, so we need to remove the operation 0018 (not the migration).